lib/bambora/client.rb

```diff
@@ -1,9 +1,15 @@
 # frozen_string_literal: true
 
+require 'base64'
+require 'excon'
+
+require 'bambora/headers'
+require 'bambora/v1/profile'
```

Comment:
I definitely take autoloading for granted. I don't really understand how it works. Testing the gem with `bin/console` works without these `require`s. However, when I tried to integrate the gem into the HiMama repo., I was receiving messages like `Uninitialized Constant Bambora::V1`, which were solved in the context of Rails by including these explicitly.

I looked through the `paper_trail` gem, to see if this architecture was common, and it seems like this is correct.

My knowledge of including files in Ruby is limited, so I would love some insight here.